roles/hyperfoil_controller/tasks/retrieve_stats.yml

```diff
@@ -15,7 +15,7 @@
   when: download_results
 - name: Download JFR
   ansible.builtin.fetch:
-    src: "{{ hyperfoil_dir }}/hfc-{{ ansible_hostname }}.jfr"
+    src: "/tmp/hyperfoil/hfc-ip-{{ ansible_default_ipv4.address | replace('.', '-') }}.jfr"
```

Comment:
The more I look at this, we probably should reuse the hyperfoil_dir like we the hyperfoil ansible scripts do and just default it to /tmp/hyperfoil